pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -2188,4 +2188,134 @@ var _ = Describe("v1beta1 inference service controller", func() {
 			}, timeout, interval).Should(BeTrue())
 		})
 	})
+	Context("If the InferenceService occured any error", func() {
```

Comment:
I didn't spot here where you are asserting the events from k8s to make sure the event was published.

Reply:
The code is modified to check if the event was published, thanks for the advice!